src/python/pcmdi/scripts/pcmdi_metrics_driver_OceanLevels.py

```diff
@@ -0,0 +1,757 @@
+#!/home/jservon/anaconda2/envs/PMP_nightly/bin/python
+#
+#
+#  USER INPUT IS SET IN FILE "input_parameters.py"
+#  Identified via --parameters key at startup
+#
+#
+import pcmdi_metrics
+import sys
+import argparse
+import os
+import json
+import genutil
+import warnings
+import cdms2
+import MV2
+import cdutil
+import collections
+import cdat_info
+import unidata
+
+# Statistical tracker
+cdat_info.pingPCMDIdb("pcmdi_metrics", "pcmdi_metrics_driver")
+
+# Before we do anything else we need to create some units
+# Salinity Units
+unidata.udunits_wrap.init()
+
+# Create a dimensionless units named dimless
+unidata.addDimensionlessUnit("dimless")
```

Comment:
Nice, we were going about this another way.. But this looks promising @dnadeau4 @doutriaux1